.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should avoid commit files under .idea folder - A nice way to achieve that is using .gitignore file